README.md

```diff
@@ -265,7 +267,15 @@ The following targets are currently supported:
 ```json
 {
   "dotnet": {
-    "namespace": "Acme.Hello"
+    /* required */
+    "namespace": "Acme.Hello",
+    "packageId": "Acme.Hello",
+
+    /* optional */
+    "title": "ACME Hello",
```

Comment:
Why do we have this? Isn't this the same as description? If it's not provided, what is the default?

Reply:
It is not--`title` is a short human-readable name, while description is more detailed. If a NuGet package has no title, it defaults to the package's `packageId`. Updated comments to reflect this.

✔️